src/fee-accounts/store/reducers/fee-accounts.reducer.ts

```diff
@@ -33,7 +37,8 @@ export function reducer(
         feeAccounts = payload.map((entity: FeeAccount) => {
             const element: FeeAccountSummary = {
               ...entity,
-              routerLink: `/fee-accounts/account/${entity.account_number}/`
+              routerLink: `/fee-accounts/account/${entity.account_number}/`,
```

Comment:
this is a sin to do something like that... u don't store links in the state... the path should be in the html template not here